vulkan/vulkan.cppm

```diff
@@ -45,10 +45,8 @@ export namespace VULKAN_HPP_NAMESPACE
   using VULKAN_HPP_NAMESPACE::ArrayProxy;
   using VULKAN_HPP_NAMESPACE::ArrayProxyNoTemporaries;
   using VULKAN_HPP_NAMESPACE::Optional;
-  using VULKAN_HPP_NAMESPACE::SharedHandle;
```

Comment:
This file is autogenerated. Could you try to find the bug in `VulkanHppGenerator.cpp` that causes `vulkan.cppm` generated in the wrong way?

Reply:
Didn't know this, I guess I'll have a look

Reply:
@theHamsta found it :)